src/emu/render.cpp

```diff
@@ -1620,6 +1629,28 @@ void render_target::load_layout_files(const internal_layout *layoutfile, bool si
 			throw emu_fatalerror("Couldn't parse default layout??");
 	}
 
+	// Check the parent of the parent to cover bios based artwork
+	if (cloneof != -1) {
```

Comment:
Please be consistent with code style - the rest of the file uses Allman style (brace on next line).

Reply:
My apologies; I will fix that.